certificate-service/integration-test/src/test/java/se/inera/intyg/certificateservice/integrationtest/CertificateReadyForSignIT.java

```diff
@@ -0,0 +1,90 @@
+package se.inera.intyg.certificateservice.integrationtest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static se.inera.intyg.certificateservice.application.certificate.dto.CertificateStatusTypeDTO.SIGNED;
+import static se.inera.intyg.certificateservice.application.testdata.TestDataCommonUserDTO.ajlaDoktorDtoBuilder;
+import static se.inera.intyg.certificateservice.application.testdata.TestDataCommonUserDTO.alvaVardadministratorDtoBuilder;
+import static se.inera.intyg.certificateservice.integrationtest.util.ApiRequestUtil.customReadyForSignCertificateRequest;
+import static se.inera.intyg.certificateservice.integrationtest.util.ApiRequestUtil.customTestabilityCertificateRequest;
+import static se.inera.intyg.certificateservice.integrationtest.util.ApiRequestUtil.defaultTestablilityCertificateRequest;
+import static se.inera.intyg.certificateservice.integrationtest.util.CertificateUtil.certificateId;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import se.inera.intyg.certificateservice.application.common.dto.AccessScopeTypeDTO;
+
+public abstract class CertificateReadyForSignIT extends BaseIntegrationIT {
+
+  protected abstract String type();
+
+  protected abstract String typeVersion();
+
+  @Test
+  @DisplayName("Vårdadministratör - Om användaren är en vårdadministratör ska intyget gå att markera som redo för signering")
```

Comment:
I think it would be great if we clearified that its an deep integrated care admin in the display name. Also, maybe instead of saying "intyget" we should refeer to it as "utkast" to better communicate the state of the certificate?

Maybe we should also add a test where the care admin is not integrated and assert that an error is thrown!